packages/components/stories/ActionDropdown.js

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
-import { // eslint-disable-line import/no-extraneous-dependencies
-	storiesOf,
-	action,
-} from '@storybook/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
prettier manages bad the comment with // eslint-disable-line. The comment move to the next line and triggers an eslint error.
https://github.com/prettier/prettier/issues/1267#issuecomment-294165170